lib/web/webidl/index.js

```diff
@@ -935,6 +936,14 @@ webidl.converters.EventHandlerNonNull = function (V) {
   return () => {}
 }
 
+webidl.attributes = {
+  Clamp: 1 << 0,
+  EnforceRange: 1 << 1,
+  AllowShared: 1 << 2,
+  AllowResizable: 1 << 3,
+  LegacyNullToEmptyString: 1 << 4
```

Comment:
can we have them in UPPERCASE letters?

Reply:
Sure, https://webidl.spec.whatwg.org/#js-extended-attributes